test/integration_test/test-deploy.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -x
 
-test_pod_template="/testspecs/operator-test-pod-template.yaml"
-test_pod_spec="/testspecs/operator-test-pod.yaml"
+test_pod_template="operator-test-pod-template.yaml"
+test_pod_spec="operator-test-pod.yaml"
```

Comment:
this is file needed?

Reply:
@pure-jliao  `operator-test-pod-template.yaml` is the template yaml spec for operator-test-pod. The `test-deploy.sh` script copy it to `operator-test-pod.yaml` and modifies all parameters and apply it on the k8s cluster. Also, what I understood from that script, it is using to delete any previous scheduled operator-test pods
```
kubectl delete -f $test_pod_template
```

Reply:
oh ok now I remember, to avoid modifying the template we added this file.